jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/PathResource.java

```diff
@@ -347,8 +347,7 @@ public long length()
         }
         catch (IOException e)
         {
-            // in case of error, use Files.size() logic of 0L
```

Comment:
The comment doesn't match the value.

Reply:
That's why I deleted the comment; I changed this implementation because all the other ones were returning -1, and I saw no reason why `PathResource` should differ.